static/docs/tutorials/index.md

```diff
@@ -0,0 +1,70 @@
+# DVC Basic Concepts and Features
+
+Learn basic concepts and features of DVC with interactive lessons:
+
+1. [Data Management](https://katacoda.com/dvc/courses/basics/data) <br/> The
+   core function of DVC is data tracking and management. Let's see how to do it.
+
+2. [Getting the Best Performance](https://katacoda.com/dvc/courses/basics/performance)
+   <br/> It is important to optimize the DVC setup for having the best
+   performance with handling big data files.
+
+3. [Tracking Data Versions](https://katacoda.com/dvc/courses/basics/versioning)
+   <br/> DVC takes advantage of GIT's versioning features to keep track of the
+   data versions.
+
+4. [Sharing Data](https://katacoda.com/dvc/courses/basics/sharing) <br/> DVC
+   facilitates sharing of data between different people that work on the same
+   project.
+
+5. [Stages And Pipelines](https://katacoda.com/dvc/courses/basics/pipelines)
+   <br/> DVC has a built-in way to connect ML steps into a DAG and run the full
+   pipeline end-to-end.
+
+# Interactive Tutorials
```

Comment:
# -> ## (it will enable the proper navigation on the right side)

Reply:
please, make basics as one single entry with and part of the Interactive Tutorials - it's too confusing to see two different sections